src/main/java/io/confluent/idesidecar/restapi/kafkarest/RecordSerializer.java

```diff
@@ -64,7 +64,7 @@ private ByteString serializeAvro(
       boolean isKey
   ) {
     try (var avroSerializer = new KafkaAvroSerializer(client)) {
-      avroSerializer.configure(serializerConfigs, isKey);
+      avroSerializer.configure(SERDE_CONFIGS, isKey);
```

Comment:
Can we use the same configs for Avro and Protobuf and JSON Schema serdes?

Reply:
Yes. I believe so. The configs in `SERDE_CONFIGS` come from the [`AbstractKafkaSchemaSerDeConfig.java`](https://github.com/confluentinc/schema-registry/blob/master/schema-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaSchemaSerDeConfig.java). This is also mentioned in a comment in the `application.yml` so that we remember not to put serializer-specific configs.

These are the configs we pass:

```
schema.registry.url=http://localhost:26636
auto.register.schemas=false
use.latest.version=false
key.subject.name.strategy=io.confluent.kafka.serializers.subject.TopicNameStrategy
value.subject.name.strategy=io.confluent.kafka.serializers.subject.TopicNameStrategy
```